compose/services/docker-compose.postgis.yml

```
@@ -0,0 +1,27 @@
+      retries: 5
+    volumes:
+      - agwise-pg:/var/lib/postgresql/data
+    networks:
+      - internal
+    environment:
+      POSTGRES_DB: agwise_db
+      POSTGRES_PASSWORD: ${PG_PASS}
+    volumes:
+    - agwise-pg:/var/lib/postgresql/data
+    networks:
+      - internal
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)